_i18n/en/resources/user-guides/remote_node_gui.md

```diff
@@ -24,24 +24,30 @@ The main menu (`Welcome to Monero` screen) will open. At the bottom left, click
 
 ![Advanced Mode](/img/resources/user-guides/en/remote_node/advanced_mode.png){:width="600px"}
 
-## Finding a public remote node
+## Finding a remote node
 
-First, you will need to find a public remote node to connect to. The website [moneroworld.com](https://moneroworld.com/#nodes) has some great resources about remote nodes, and the website [monero.fail](https://monero.fail) has a list of functioning remote nodes. 
+First, you will need to find a node to connect to.
 
-## Configuring your wallet to connect to a custom public remote node
+It is important to note that there are publicly advertised remote nodes which are run by malicious actors with the sole purpose of deanonymizing users.
```

Comment:
"deanonymizing users"  -> lets expand/clarify this.

```suggestion
It is important to note that there are publicly advertised remote nodes which are run by malicious actors with the sole purpose of deanonymizing users/outputs by, for example:
- Linking clearnet IP addresses to txid's. 
- Providing bogus decoys to determine which output is the real spend. 
- For more information see [YouTube - Breaking Monero Episode 07: Remote Nodes](https://www.youtube.com/watch?v=n6Bxp0k7Uqg)

Reply:
Done

Reply:
- Providing bogus decoys to determine which output is the real spend.  > determining the source of coins

can be better

Reply:
> Linking ~clearnet~ IP addresses to TX IDs

clearnet is fine imo, even if the reader glosses over it, they need to be introduced to such terminology, especially when Tor/I2p is mentioned later.

>Providing bogus decoys to determine which output is the real spend -> determining the source of coins

This is a good change yes

should we mention only the goal (not the method)?

"Attempt to determine the source of coins" 

Or hinting at both the method and goal:
"Bogus decoys are provided in an attempt to..^"

If we mention only the goal, users might be more inclined to click on the linked video.

Reply:
The original is accurate

Reply:
also for @clearnet https://www.getmonero.org/resources/moneropedia/clearnet.html 

The remote node episode in the breaking monero series is important to share / introduce. stating only "deanon users" is vague / broad and is now the perfect learning opportunity.after grabbing the readers attention.

Reply:
At the very least, the youtube video needs to be linked to clarify the vague 'sole purpose of deanonymizing users' statement, readers will definitely want to know more about that.

Reply:
I added the playlist already,i think